filebeat/harvester/log.go

```diff
@@ -87,6 +88,11 @@ func (h *Harvester) Harvest() {
 		if !h.sendEvent(event) {
 			return
 		}
+
+		if h.config.CloseTimeout > 0 && time.Since(startTime) > h.config.CloseTimeout {
+			logp.Info("Closing harvester because ttl was reached: %s", h.path)
+			return
+		}
```

Comment:
file reader has some 'inactive timeout'? The 'inactive timeout' should be <= `close_timeout`.

What about multiline timeout? hm...